src/globalevent.cpp

```diff
@@ -263,21 +275,10 @@ bool GlobalEvent::configureEvent(const pugi::xml_node& node)
 
 		nextExecution = current_time + difference;
 		eventType = GLOBALEVENT_TIMER;
-	} else if ((attr = node.attribute("type"))) {
-		const char* value = attr.value();
-		if (strcasecmp(value, "startup") == 0) {
-			eventType = GLOBALEVENT_STARTUP;
-		} else if (strcasecmp(value, "shutdown") == 0) {
-			eventType = GLOBALEVENT_SHUTDOWN;
-		} else if (strcasecmp(value, "record") == 0) {
-			eventType = GLOBALEVENT_RECORD;
-		} else {
-			std::cout << "[Error - GlobalEvent::configureEvent] No valid type \"" << attr.as_string() << "\" for globalevent with name " << name << std::endl;
-			return false;
-		}
 	} else if ((attr = node.attribute("interval"))) {
 		interval = std::max<int32_t>(SCHEDULER_MINTICKS, pugi::cast<int32_t>(attr.value()));
 		nextExecution = OTSYS_TIME() + interval;
+		eventType = GLOBALEVENT_TIMER;
```

Comment:
Are you sure about this change? 😄

Reply:
Pretty much! Otherwise, it is registered as `onThink` which makes no sense for interval events...

(somewhat late, this response...)

Reply:
If interval can no longer be onThink then how do we make an onThink event from xml?